psi4/driver/driver.py

```diff
@@ -679,6 +679,9 @@ def gradient(name, **kwargs):
     return_wfn = kwargs.pop('return_wfn', False)
     core.clean_variables()
 
+    if isinstance(lowername, str) and lowername.replace('-', '') == 'mp2d':
+        raise ValidationError('MP2D gradients (even findif) not available.')
```

Comment:
Out of curiosity, is there are reason for this, or just not yet implemented?

Reply:
I was mostly working on this in November, and while I got the correction gradients and the analytic(mp2 + correction) gradients to pass, they never matched the findif(mp2 + correction) gradients. And I was making the reference values, too, so doubly shifty. So I only copied the energy code over. I'm pretty confident on that, as Chandler and I exchanged some refs. Until I get a chance to carefully inspect who's going wrong, I wanted the grads marked with a minefield.